helm-charts/falcon-sensor/templates/node_scc.yaml

```diff
@@ -31,7 +31,7 @@ requiredDropCapabilities:
 defaultAddCapabilities:
 allowedCapabilities:
 users:
-- crowdstrike-falcon-sa
+- {{include "falcon-sensor.serviceAccountName" .}}
```

Comment:
This should be `{{ .Values.serviceAccount }}`

Reply:
done